pepper-apis/dsm-core/src/main/java/org/broadinstitute/dsm/util/NotificationUtil.java

```diff
@@ -80,7 +82,10 @@ public static Long getLastChanged(@NonNull String participantId) {
 
     public synchronized void startup(@NonNull Config config) {
         emailKey = config.getString(ApplicationConfigConstants.EMAIL_KEY);
-        emailClientSettings = (JsonObject) (new JsonParser().parse(config.getString(ApplicationConfigConstants.EMAIL_CLIENT_SETTINGS)));
+        emailProxyUrl = config.getString(ApplicationConfigConstants.EMAIL_PROXY_URL);
+        JsonObject senderInfo = (JsonObject) (new JsonParser().parse(config.getString(ApplicationConfigConstants.EMAIL_CLIENT_SETTINGS)));
+        emailSender = new EmailSender(senderInfo.get("sendGridFrom").getAsString(), senderInfo.get("sendGridFromName").getAsString());
```

Comment:
Nit: You can skip the JsonObject stuff by deserializing `EMAIL_CLIENT_SETTINGS` directly into `EmailSender`, like this: `Gson().fromJson(ApplicationConfigConstants.EMAIL_CLIENT_SETTINGS, EmailSender.class)`.

Reply:
I think it's a little different:
`new Gson().fromJson(config.getString(ApplicationConfigConstants.EMAIL_CLIENT_SETTINGS), EmailSender.class);`

And for good measure, I'll add `@SerializedName` annotations to `EmailSender`